### PR TITLE
Add deserialization

### PR DESCRIPTION
**Pull request by joeljames**
@petebrowne @ecordell @reneric @thatgibbyguy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,12 +53,53 @@ class CommentSerializer(Serializer):
...
+            'content',
+            'created_date',
+            'created_time',
+        )
+
+# The dictionary data to be serialized
+data_dict = {'user': {'email': 'foo@example.com', 'username': 'JohnSmith'}, 'content': 'foo bar', 'created_date': '2015-01-01', 'created_time': '2012-01-01T16:00:00Z'}
+
+deserializer = CommentDeserializer(data_dict=data_dict)
+deserializer.objects.user.username
```
Comment (petebrowne) [inline README.md]: It seems weird to use a plural word `objects` to access a single object

Comment (joeljames) [inline README.md]: Makes sense. I will update it to just `object`

**Comment by petebrowne** [comment]
:+1:

**Comment by ecordell** [comment]
:+1:

**Comment by joeljames** [comment]
Build is green (https://travis-ci.org/localmed/pyserializer/branches). I have added functional tests, and also manually tested the deserialization feature. Merging this in and pushing up a new release(0.1.0) on pypi.

**Pull request merged**